From 9a752163633a8b8d26d2d3be082ef1da8767ee81 Mon Sep 17 00:00:00 2001
From: Vladimir Oltean <olteanv@gmail.com>
Date: Tue, 3 Dec 2019 11:56:08 +0200
Subject: [PATCH buildroot 05/18] net: dsa: sja1105: give the SPI kernel thread
 real-time scheduling class

To be fair, we can do this from user space as well, by finding $(pidof
ptp1), but since the facilities are already there in the SPI core, just
use those.

Signed-off-by: Vladimir Oltean <olteanv@gmail.com>
---
 drivers/net/dsa/sja1105/sja1105_main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/dsa/sja1105/sja1105_main.c b/drivers/net/dsa/sja1105/sja1105_main.c
index de9e4b6a833b..5b579e6daccc 100644
--- a/drivers/net/dsa/sja1105/sja1105_main.c
+++ b/drivers/net/dsa/sja1105/sja1105_main.c
@@ -2081,6 +2081,7 @@ static int sja1105_probe(struct spi_device *spi)
 
 	/* Configure the SPI bus */
 	spi->bits_per_word = 32;
+	spi->rt = true;
 	rc = spi_setup(spi);
 	if (rc < 0) {
 		dev_err(dev, "Could not init SPI\n");
-- 
2.17.1

